Kernel/Arch/x86/TrapFrame.h

```diff
@@ -6,6 +6,9 @@
 
 #pragma once
 
+#include <AK/Platform.h>
+VALIDATE_IS_X86()
```

Comment:
nitpick: Can we move these after all of the includes? I think that would fit more with the serenity style.